Spokestack/NLUDelegate.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NLUDelegate.swift
+//  Spokestack
+//
+//  Created by Noel Weichbrodt on 2/14/20.
+//  Copyright © 2020 Pylon AI, Inc. All rights reserved.
```

Comment:
Ping again on the copyright string in the new file template :)

Reply:
Ah yes. Changed the template, I will find-replace the whole project after merge.